client/my-sites/stats/controller.jsx

```diff
@@ -301,11 +301,10 @@ export function summary( context, next ) {
 	const extraProps =
 		context.params.module === 'videodetails' ? { postId: parseInt( queryOptions.post, 10 ) } : {};
 
-	let statsQueryOptions = {};
+	// The option is used for stats queries only.
+	const statsQueryOptions = pick( queryOptions, [ 'num', 'summarize', 'geoMode' ] );
 
-	// All Time Summary Support
-	if ( queryOptions.summarize && queryOptions.num ) {
```

Comment:
We may still need the `summarize` and `num` checks simultaneously, or it would be broken when the number of days is unset.

<img width="1367" alt="截圖 2025-05-08 下午4 16 10" src="https://github.com/user-attachments/assets/f0f0e6e4-1d09-4a84-9b0e-abcf2e74fcc0" />